packages/cubejs-databricks-jdbc-driver/src/installer.ts

```diff
@@ -15,23 +17,32 @@ function acceptedByEnv() {
   return acceptStatus;
 }
 
+/**
+ * In the beginning of 2025 Databricks released their open-source version of JDBC driver and encourage
+ * all users to migrate to it as company plans to focus on improving and evolving it over legacy simba driver.
+ * More info about OSS Driver could be found at https://docs.databricks.com/aws/en/integrations/jdbc/oss
+ * As of March 2025 To use the Databricks JDBC Driver (OSS), the following requirements must be met:
+ * Java Runtime Environment (JRE) 11.0 or above. CI testing is supported on JRE 11, 17, and 21.
+ */
 export async function downloadJDBCDriver(): Promise<string | null> {
   const driverAccepted = acceptedByEnv();
 
   if (driverAccepted) {
```

Comment:
When I first added the Data Bricks driver, it had its own license, so I added license accept to protect potential lawyer issues. 

From the first look, I assume that the new OSS JDBC driver is Apache 2. Right?

If yes, let's remove the code for acceptance. It's not needed anymore.

Reply:
Yeah, new driver is `Apache 2.0`. I'll remove the acceptance check/code/env.